pipeline/inputs/nginx.md

```diff
@@ -130,8 +130,7 @@ Fluent Bit v2.x.x
 
 ## Exported Metrics
 
-This documentation is copied from the nginx prometheus exporter metrics documentation: 
-[https://github.com/nginxinc/nginx-prometheus-exporter/blob/master/README.md].
+This documentation is copied from the [nginx prometheus exporter metrics documentation](https://github.com/nginxinc/nginx-prometheus-exporter/blob/main/README.md).
```

Comment:
Updated the capitalization to match the product name and page title, and indicated that the page is located on GitHub (to let users know they're leaving the docs).

```suggestion
This documentation is copied from the
[NGINX Prometheus Exporter metrics documentation](https://github.com/nginxinc/nginx-prometheus-exporter/blob/main/README.md)
on GitHub.
```

Reply:
Thanks!